src/test/java/com/commercetools/sync/shoppinglists/utils/LineItemListUpdateActionUtilsTest.java

```diff
@@ -0,0 +1,559 @@
+import static java.util.stream.Collectors.toList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class LineItemListUpdateActionUtilsTest {
+
+    private static final String RES_ROOT = "com/commercetools/sync/shoppinglists/utils/lineitems/";
+    private static final String SHOPPING_LIST_WITH_LINE_ITEMS_SKU_123 =
+        RES_ROOT + "shoppinglist-with-lineitems-sku-123.json";
+    private static final String SHOPPING_LIST_WITH_LINE_ITEMS_SKU_123_WITH_CHANGES =
+        RES_ROOT + "shoppinglist-with-lineitems-sku-123-with-changes.json";
+    private static final String SHOPPING_LIST_WITH_LINE_ITEMS_SKU_12 =
+        RES_ROOT + "shoppinglist-with-lineitems-sku-12.json";
+    private static final String SHOPPING_LIST_WITH_LINE_ITEMS_SKU_1234 =
+        RES_ROOT + "shoppinglist-with-lineitems-sku-1234.json";
+    private static final String SHOPPING_LIST_WITH_LINE_ITEMS_SKU_124 =
+        RES_ROOT + "shoppinglist-with-lineitems-sku-124.json";
+    private static final String SHOPPING_LIST_WITH_LINE_ITEMS_SKU_312 =
+        RES_ROOT + "shoppinglist-with-lineitems-sku-312.json";
+    private static final String SHOPPING_LIST_WITH_LINE_ITEMS_SKU_32 =
+        RES_ROOT + "shoppinglist-with-lineitems-sku-32.json";
+    private static final String SHOPPING_LIST_WITH_LINE_ITEMS_SKU_1324 =
+        RES_ROOT + "shoppinglist-with-lineitems-sku-1324.json";
+    private static final String SHOPPING_LIST_WITH_LINE_ITEMS_SKU_1423 =
+        RES_ROOT + "shoppinglist-with-lineitems-sku-1423.json";
+    private static final String SHOPPING_LIST_WITH_LINE_ITEMS_SKU_324 =
+        RES_ROOT + "shoppinglist-with-lineitems-sku-324.json";
+    private static final String SHOPPING_LIST_WITH_LINE_ITEMS_SKU_132_WITH_CHANGES =
+        RES_ROOT + "shoppinglist-with-lineitems-sku-132-with-changes.json";
+    private static final String SHOPPING_LIST_WITH_LINE_ITEMS_NOT_EXPANDED =
+        RES_ROOT + "shoppinglist-with-lineitems-not-expanded.json";
+    private static final String SHOPPING_LIST_WITH_LINE_ITEMS_SKU_123_WITH_DIFFERENT_ADDED_AT =
+        RES_ROOT + "shoppinglist-with-lineitems-sku-123-with-different-addedAt.json";
+    private static final String SHOPPING_LIST_WITH_LINE_ITEMS_SKU_123_WITHOUT_ADDED_AT =
+        RES_ROOT + "shoppinglist-with-lineitems-sku-123-without-addedAt.json";
+
+    private static final ShoppingListSyncOptions SYNC_OPTIONS =
+        ShoppingListSyncOptionsBuilder.of(mock(SphereClient.class)).build();
+
+    private static final LineItemReferenceResolver lineItemReferenceResolver =
+        new LineItemReferenceResolver(SYNC_OPTIONS, getMockTypeService());
+
+    @Test
+    void buildLineItemsUpdateActions_WithEmptyNewAndNullOldLineItems_ShouldNotBuildActions() {
```

Comment:
Is it the same as this test case?
`void buildLineItemsUpdateActions_WithNullOldAndEmptyNewLineItems_ShouldNotBuildActions()`

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/618/commits/a7a7e0a30b50ef509dd70e39f0b9a6e9ca9e79d8